src/haxelib/server/SiteDb.hx

```diff
@@ -145,11 +151,25 @@ class SiteDb {
 			Project.manager,
 			Tag.manager,
 			Version.manager,
-			Developer.manager
+			Developer.manager,
+			Meta.manager
 		];
-		for (m in managers)
-			if (!TableCreate.exists(m))
+
+		var hasOldTables = false;
+
+		for (m in managers) {
+			if (TableCreate.exists(m)) {
+				hasOldTables = true;
+			} else {
 				TableCreate.create(m);
```

Comment:
@andyli Regarding what you said about multiple server instances, this may be a problem here since checking if a table exists and creating it is not done atomically. I'm not sure if this might end up being an issue if two instances both add conflicting `Meta` table entries, which is meant to be a table with a single field? If the Table.create would fail in that case anyway then I guess it wouldn't be a problem.

Reply:
Yeah. I think the latter command will fail when there are two parallel `TableCreate.create()`. Ideally we should use `CREATE TABLE IF NOT EXISTS`... but well.

I agree it should be fine to let the latter one fail since apache/k8s will restart the instance.